05-Jason-Luo/wk02/5-fri/atm/js/atm.js

```diff
@@ -0,0 +1,199 @@
+//main objectives
+//deposit money
+//withdraw money
+//cant withdraw if not enough money for account balance
+//overdraft protection
+
+//deposit 
+//connect buttons to functions
+function checkForZero() {
+    $('.zero').removeClass('zero');
+    let checkingMoney = ($("#checking-balance").text());
+    console.log(checkingMoney);
+    let initMoney = parseInt(checkingMoney.substring(1, checkingMoney.length));
+    if (initMoney <= 0) {
+        $('#checking-balance').addClass('zero');
+    }
+
+    let savingsMoney = ($("#savings-balance").text());
+    console.log(savingsMoney);
+    let overallSavings = parseInt(savingsMoney.substring(1, savingsMoney.length));
+    if (overallSavings <= 0) {
+        $('#savings-balance').addClass('zero');
+    }
+
+}
+
+
+$(document).ready(function () {
+    checkForZero();
+    let checkingAccount = { balance: 2320 };
+    let savingAccount = { balance: 23255 };
+
+    function update() {
+        let checkBal = $("#checking-balance").html("$" + checkingAccount.balance);
+        console.log(checkBal);
+
+        let savBal = $("#savings-balance").html("$" + savingAccount.balance);
+        console.log(savBal);
+    }
+
+    update();
+
+    function depositMoney() {
+        $("#checking-deposit").on('click', function () {
+            let initCheck = ($("#checking-balance").text());
+            console.log(initCheck);
+            let initMoney = parseInt(initCheck.substring(1, initCheck.length));
+            console.log(initMoney);
+            let deposit = parseInt($("#checking-amount").val());
+            console.log(deposit);
+            let total = deposit + initMoney;
+            console.log(total);
+            $('#checking-balance').text('$' + total);
+            checkForZero();
+        });
+    };
+
+    depositMoney();
```

Comment:
```suggestion

    function accountDepositListener(account) {
        $(`#${account}-deposit`).on( 'click', () => {
            const $balance = $(`#${account}-balance`);

            let initCheck = $balance.text();
            let initMoney = parseInt(initCheck.substring(1));
            let deposit = parseInt($(`#${account}-amount`).val());
            let total = deposit + initMoney;

            $balance.text('$' + total);

            checkForZero();
        });
    };

    accountDepositListener('checking');
    accountDepositListener('savings');
```
Here are some generic functions that deal with depositing money. You wouldn't need your `.savingsDeposit()` function either this way.

Also note that the second argument of the `.substring()` method will default to the final character, so you can leave it out in this case.